src/drivers/windows/lv_windows_display.c

```diff
@@ -143,6 +209,8 @@ static unsigned int __stdcall lv_windows_display_thread_entrypoint(
         return 0;
     }
 
+    LV_ASSERT(lv_windows_center_window(window_handle));
```

Comment:
You need to mention this behavior change in the comment of lv_windows_create_display API.